app/utils/FormMappings.scala

```diff
@@ -51,10 +51,20 @@ object FormMappingsConstants {
 @Singleton
 class FormMappings @Inject() (val messagesApi: MessagesApi) extends PayrollBikDefaults with I18nSupport {
 
-  private val nameValidationRegex = "([a-zA-Z-'\\sôéàëŵŷáîïâêûü])*"
-  private val ninoValidationRegex = "([a-zA-Z])([a-zA-Z])[0-9][0-9][0-9][0-9][0-9][0-9]([a-zA-Z]?)"
-  private val ninoTrimmedRegex    = "([a-zA-Z])([a-zA-Z])[0-9][0-9][0-9][0-9][0-9][0-9]"
-  private val maxLength           = 100
+  private val nameValidationRegex        = "([a-zA-Z-'\\sôéàëŵŷáîïâêûü])*"
+  private val ninoValidationRegex        = "([a-zA-Z])([a-zA-Z])[0-9][0-9][0-9][0-9][0-9][0-9]([a-zA-Z]?)"
+  private val ninoTrimmedRegex           = "([a-zA-Z])([a-zA-Z])[0-9][0-9][0-9][0-9][0-9][0-9]"
+  private val dateDayRegex: String       = "([0-9])"
+  private val dateMonthRegex: String     = "([0-9])"
+  private val dateYearRegex: String      = "([0-9]){4}"
```

Comment:
can we remove the type annotations here? Just to keep it consistent with the others and we can easily see it's a String.